plugins/modules/networkfirewall.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: networkfirewall
+short_description: manage AWS Network Firewall firewalls
+version_added: 4.0.0
+description:
+  - A module for creating, updating and deleting AWS Network Firewall firewalls.
+options:
+  arn:
+    description:
+      - The ARN of the firewall.
+      - Exactly one of I(arn) or I(name) must be provided.
+    required: false
+    type: str
+  name:
```

Comment:
can we also add an alias `firewall_name`, because the output is using it.

Reply:
sure